cli-output/src/display.rs

```diff
@@ -5,9 +5,12 @@ use {
     indicatif::{ProgressBar, ProgressStyle},
     solana_sdk::{
         clock::UnixTimestamp, hash::Hash, message::Message, native_token::lamports_to_sol,
-        program_utils::limited_deserialize, transaction::Transaction,
+        program_utils::limited_deserialize, pubkey::Pubkey, transaction::Transaction,
     },
     solana_transaction_status::UiTransactionStatusMeta,
+    spl_memo_v1::id as spl_memo_v1_id,
+    spl_memo_v2::id as spl_memo_v2_id,
+    spl_memo_v3::id as spl_memo_v3_id,
```

Comment:
```suggestion
    spl_memo::v1::id as spl_memo_v1_id,
    spl_memo::id as spl_memo_v3_id,
```
v2 of SPL memo was a purely solana-sdk release; the program ID didn't change. So we don't need to handle it separately